Sources/LiveKit/Protocols/VideoRenderer.swift

```diff
@@ -27,4 +28,39 @@ public protocol VideoRenderer: RTCVideoRenderer {
     /// This will be invoked on the .main thread.
     @objc
     var adaptiveStreamSize: CGSize { get }
+
+    /// Size of the frame.
+    func set(size: CGSize)
+
+    func render(frame: VideoFrame)
+}
+
+class VideoRendererAdapter: NSObject, LKRTCVideoRenderer {
```

Comment:
how is this new class used? is it worth updating README or some embedded comments on usage?

Reply:
This is used only internally, I will mark it explicity.